### ISSUE TITLE
Connection requests on the same hub are delayed by a few hours

### ISSUE THREAD

**Issue by jeroenpraat** [redmatrix/hubzilla#150]
From https://hubzilla.nl/display/ba59c885afdc0bc7f4192088129814c33d9e6ad1efd6da6c6de961648307f959@red.mariovavti.com

Jeroen:
> the connection requests started to arrive eventually. 1.5 hour to late, but they still arrived. I am wondering how this can happen. Has this maybe to do with the fact redmatrix.nl is also running on my VPS? Not that I have a high load average, the contrary.
> P.S. Just received another connection request. So that's 5 hours to late. 

Mario:
> i see the same behaviour on my hub... not sure what's going on but there is probably an issue... notifications between channels on the same hub used to be almost instant...

**Comment by kenrestivo** [comment]
Are the API calls 500'ing?

If so, and if you have commit access, can you merge this in? 
https://github.com/redmatrix/hubzilla/pull/144

**Comment by kenrestivo** [comment]
Yep, that's it.
```sh
curl -i -ssl -u test:test https://hubzilla.nl/api/statuses/friends_timeline.json
```
Returns 500.

Please do a git pull. And accept my apologies.

**Comment by jeroenpraat** [comment]
This did not fix it. Still waiting on the connection request after doing a new test.

@redmatrix  I understand you can't confirm it, but @git-marijus can.

**Comment by jeroenpraat** [comment]
The connection request now arrived in 25 minutes. That's an approvement, but still not less than few seconds as before.

**Comment by royalterra** [comment]
Yeah Still is the seam problem to me 2 hours and the queue become from 1700 to now 5000 .

**Comment by kenrestivo** [comment]
@jeroenpraat  Was this fixed by https://github.com/redmatrix/hubzilla/commit/391807b46c3f54eb94b88c253d89d09bc7139989 ?

**Comment by jeroenpraat** [comment]
@kenrestivo Unfortunately not. I did two test with newly created channel and it took 30 min. and 25 min. before the request arrived.

**Comment by royalterra** [comment]
In the seam server 2 diiferent hubs say to me 2 hours some time 30 minutes

**Comment by git-marijus** [comment]
so far i can say it works normal between two hubs...

**Comment by git-marijus** [comment]
This is the log when the new connection notification appeared:

2015-11-14 15:14:01: onepoll.php:16:onepoll_run: onepoll: start
2015-11-14 15:14:02:bbhoebmpeul975gis36ua0d675 zot.php:495:zot_refresh: New introduction received for test-4
2015-11-14 15:14:02:bbhoebmpeul975gis36ua0d675 enotify.php:441:notification: notification: sending notification email
2015-11-14 15:14:03:bbhoebmpeul975gis36ua0d675 zot.php:2869:build_sync_packet: build_sync_packet
2015-11-14 15:14:04: onepoll.php:16:onepoll_run: onepoll: start

**Comment by redmatrix** [comment]
probably fixed in 1029939f

**Comment by jeroenpraat** [comment]
Yes! Not only local forums, but connection requests to all local channels where delayed. That's why there didn't rang a bell when I saw that commit. Closing and thanks.
